### PR TITLE
Redux integrated with products loading and page handling.

### PR DESCRIPTION
**Pull request by omiearicent**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```
Comment (sgupta-aricent) [inline src/App.js:1]: update file name to app.js

File: src/App.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 import './App.css';
-import ProductDashBoard from './components/Products/ProductDashBoard/ProductDashBoard'
+import ProductDashBoard from './container/productDashBoard/ProductDashBoard'
```
Comment (sgupta-aricent) [inline src/App.js]: use camel casing for file names

File: src/components/Products/ProductTile/ProductTile.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+
     return (<div className={props.customClass}>
         {ProductRender(props.products)}
     </div>)
 
 }
 
-export default ProductTile
\ No newline at end of file
+export default productTile
```
Comment (sgupta-aricent) [inline src/components/Products/ProductTile/ProductTile.js]: Class name should be ProductTile but file name should be productTile.js

File: src/store/actions/actionConstants.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export  const PRODUCT_LOAD='PRODUCT_LOAD'
+export const PAGE_LOAD = 'PAGE_LOAD'
```
Comment (sgupta-aricent) [inline src/store/actions/actionConstants.js:2]: format this page

File: src/store/reducers/activePage.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import * as actionTypes from '../actions/actionConstants'
```
Comment (sgupta-aricent) [inline src/store/reducers/activePage.js]: update page name to activePageReducer.js

File: src/components/Products/ProductTile/ProductTile.js [hunk 6]
```diff
@@ -1,21 +1,21 @@
...
-function ProductTile(props) {
+function productTile(props) {
     let ProductRender = (products) => {
         return products.map(product => <div className="product"><a href="#">
             <img src={product.ImgUrl} alt="" width="100" height="100" /></a>
             <div className="product-description">
-                       Name : {product.Name}
+                Name : {product.Name}
                 <br />Brand : {product.Brand}
                 <br />Price : {product.Price}
```
Comment (sgupta-aricent) [inline src/components/Products/ProductTile/ProductTile.js]: use string interpolation

**Review by sgupta-aricent — CHANGES_REQUESTED** [review]
Use proper naming conventions and file names

File: src/App.js (hunk 1 above)
Comment (omiearicent) [reply src/App.js:1]: Renamed.

File: src/components/Products/ProductTile/ProductTile.js (hunk 3 above)
Comment (omiearicent) [reply src/components/Products/ProductTile/ProductTile.js]: changed to ProductTile

File: src/store/actions/actionConstants.js (hunk 4 above)
Comment (omiearicent) [reply src/store/actions/actionConstants.js:2]: done.

File: src/store/reducers/activePage.js (hunk 5 above)
Comment (omiearicent) [reply src/store/reducers/activePage.js]: Updated.

File: src/components/Products/ProductTile/ProductTile.js (hunk 6 above)
Comment (omiearicent) [reply src/components/Products/ProductTile/ProductTile.js]: Implemented.

File: src/App.js (hunk 2 above)
Comment (omiearicent) [reply src/App.js]: Done.

File: src/store/actions/action.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
+import { PRODUCT_LOAD, PAGE_LOAD } from '../actions/actionConstants'
+
+export const getProduct = (name) => {
+    return function (dispatch) {
+        axios.get('/res/products.json')
+            .then(res => {
+                const products = res.data;
+                dispatch({ type: PRODUCT_LOAD, state: { data: products } });
```
Comment (sgupta-aricent) [inline src/store/actions/action.js]: remove state parameter

**Review by sgupta-aricent — CHANGES_REQUESTED** [review]

File: src/store/actions/action.js (hunk 7 above)
Comment (omiearicent) [reply src/store/actions/action.js]: done.

**Review by sgupta-aricent — APPROVED** [review]
changes look good

**Pull request merged**